view/statistics-flow-operators.js

```diff
@@ -0,0 +1,161 @@
+		}
+		serviceToCertLegacyMatch[getServiceName(ServiceType)].push('certificate-' + certificate.key);
+		serviceToCertLegacyMatch[''].push('certificate-' + certificate.key);
+	});
+});
+
+exports['statistics-main'] = function () {
+	var queryHandler, tables = new ObservableValue({})
+	  , pagination = new Pagination('/flow/by-operator/'), tablesValue = {};
+
+	Object.keys(processingSteps).forEach(function (stepShortPath) {
+		tablesValue[stepShortPath] = new ObservableValue({});
+	});
+
+	tables.value = tablesValue;
+
+	queryHandler = setupQueryHandler(queryHandlerConf,
+		location, '/flow/by-operator/');
+
+	queryHandler.on('query', function (query) {
+		var serverQuery = copy(query), dateFrom, dateTo;
+
+		dateFrom = query.dateFrom;
+		dateTo   = query.dateTo || new db.Date();
+
+		serverQuery.dateFrom = dateFrom.toJSON();
+		serverQuery.dateTo = dateTo.toJSON();
+		// hard code for tests
+
+		queryServer(serverQuery).done(function (responseData) {
+			tables.value[query.step].value = responseData.data;
+
+			pagination.current.value = Number(serverQuery.page);
+			pagination.count.value   = responseData.pageCount;
+		});
+	});
+
+	section({ class: 'section-primary users-table-filter-bar' },
+		form({ action: '/flow/by-operator/', autoSubmit: true },
+			div({ class: 'users-table-filter-bar-status' },
+				selectService({ label: _("All services") })),
+			div({ class: 'users-table-filter-bar-status' },
+				selectCertificate(),
+				legacy('selectMatch', 'service-select', serviceToCertLegacyMatch)),
+			div({ class: 'users-table-filter-bar-status' },
+				selectUser({ label: _("All operators"), name: 'processor',
+					usersCollection: usersCollection })),
+			div(
+				{ class: 'users-table-filter-bar-status' },
+				label({ for: 'date-from-input' }, _("Date from"), ":"),
+				input({ id: 'date-from-input', type: 'date',
+					name: 'dateFrom', value: location.query.get('dateFrom').map(function (dateFrom) {
+					var now = new db.Date(), defaultDate;
+					defaultDate = new db.Date(now.getUTCFullYear(), now.getUTCMonth(),
+								now.getUTCDate() - 6);
+
+					return dateFrom || defaultDate;
+				}) })
+			),
+			div(
+				{ class: 'users-table-filter-bar-status' },
+				label({ for: 'date-to-input' }, _("Date to"), ":"),
+				input({ id: 'date-to-input', type: 'date',
+					name: 'dateTo', value: location.query.get('dateTo') })
+			),
+			select(
+				{ id: 'select-step', name: 'step' },
+				list(Object.keys(processingSteps), function (step) {
+					return option({
+						value: step,
+						selected: location.query.get('step').map(function (value) {
+							var selected = (step ? (value === step) : (value == null));
+							return selected ? 'selected' : null;
+						})
+					}, getStepLabelByShortPath(step));
+				})
+			),
+			selectPeriodMode(),
+			p({ class: 'submit' }, input({ type: 'submit' }))));
+
+	section(pagination);
+	section({ class: "section-primary" },
+		location.query.get('step').map(function (key) {
```

Comment:
`step` may not be in `query` and we should not show empty page in such case but first step.

We should draw it same way as other views, where we have one observable object, which is updated whenever query changes, and it's change invokes table re-render.

Relying on query observables as above is error-prone (we already observed that in previous views)